resources/views/app.blade.php

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
-<html lang="{{ App::currentLocale() }}">
+<html lang="{{ App::currentLocale() }}" dir="auto">
```

Comment:
The reason that we will be setting this in body rather than HTML is because the "auto" value looks into the text content of the element to determine the directionality. See https://www.w3.org/International/questions/qa-html-dir#dirauto